addons/mass_mailing/models/mailing.py

```diff
@@ -472,6 +473,16 @@ def _compute_ab_testing_description(self):
                 mailing._get_ab_testing_description_values()
             )
 
+    @api.depends('campaign_id.mailing_mail_ids.state')
+    def _compute_is_ab_test_sent(self):
+        for rec in self:
+            ab_testing_mailings = rec._get_ab_testing_siblings_mailings().sudo()
```

Comment:
Why a sudo ?

Reply:
Hello

I'm also not sure why it was added as I tried removing it and works fine (Functionally I only know a single flow, so it would be great if you give it a spin).

Thanks

Reply:
Well there is no reason you have access to a given mailing and not its siblings, so well I would not be stressed about that ^^ .